docs/api/migration-guides/qiskit-1.0-features.mdx

```diff
@@ -0,0 +1,1066 @@
+
+
+The following examples show the migration paths of the three simulators in `basicaer`.
+
+
+<Tabs>
+  <TabItem value="statevec" label="Statevector simulator" default>
+    ```python
+    from qiskit import QuantumCircuit
+
+    qc = QuantumCircuit(3)
+    qc.h(0)
+    qc.h(1)
+    qc.cx(1, 2)
+    qc.measure_all()
+
+    # Old
+    from qiskit import BasicAer
+    backend = BasicAer.get_backend("statevector_simulator")
+    statevector = backend.run(qc).result().get_statevector()
+
+    # Alternative
+    qc.remove_final_measurements()  # no measurements allowed
+    from qiskit.quantum_info import Statevector
+    statevector = Statevector(qc)
+    ```
+  </TabItem>
+  <TabItem value="unitary" label="Unitary simulator">
+    ```python
+    from qiskit import QuantumCircuit
+    qc = QuantumCircuit(3)
+    qc.h(0)
+    qc.h(1)
+    qc.cx(1, 2)
+    qc.measure_all()
+
+    # Old
+    from qiskit import BasicAer
+    backend = BasicAer.get_backend("unitary_simulator")
+    result = backend.run(qc).result()
+
+    # Alternative
+    qc.remove_final_measurements()  # no measurements allowed
+    from qiskit.quantum_info import Operator
+    result = Operator(qc).data
+    ```
+  </TabItem>
+  <TabItem value="qasm" label="QASM simulator">
+    ```python
+    from qiskit import QuantumCircuit
+    qc = QuantumCircuit(3)
+    qc.h(0)
+    qc.h(1)
+    qc.cx(1, 2)
+    qc.measure_all()
+
+    # Old
+    from qiskit import BasicAer
+    backend = BasicAer.get_backend("qasm_simulator")
+    result = backend.run(qc).result()
+
+    # Alternative
+    from qiskit.providers.basic_provider import BasicProvider
+    backend = BasicProvider().get_backend("basic_simulator")
+    result = backend.run(qc).result()
+
+    # or, directly
+    from qiskit.providers.basic_provider import BasicSimulator
+    backend = BasicSimulator()
+    result = backend.run(qc).result()
+    ```
+  </TabItem>
+</Tabs>
+
+<span id="fake-backends-runtime"></span>
+### fake_provider
+
+Most of the user-facing components of
+`qiskit.providers.fake_provider` have
+been migrated to the `qiskit-ibm-runtime` Python package. This includes the
+fake provider classes, all of the device-specific fake backends (such as
+`FakeVigo`, `FakeNairobiV2`, `FakeSherbrooke` and so on), and the fake backend
+base classes. Click through the following tabs to see the affected classes.
+
+<Tabs>
+  <TabItem value="backends" label="Fake Backends" default>
+      * Any class in `qiskit.providers.fake_provider.backends`
+      * `fake_provider.fake_backend.FakeBackend`
+      * `fake_provider.fake_backend.FakeBackendV2`
+  </TabItem>
+  <TabItem value="providers" label="Fake Providers">
+      * `fake_provider.FakeProvider`
+      * `fake_provider.FakeProviderForBackendV2`
+      * `fake_provider.FakeProviderFactory`
+  </TabItem>
+</Tabs>
+
+To migrate to the new path:
+
+1. Install a version of `qiskit-ibm-runtime` higher than `0.17.1`:
```

Comment:
```suggestion
1. Install `qiskit-ibm-runtime` `0.17.1` or later:
```

Reply:
The original text says "higher than", but the code says >= , so I went with .17.1 or later.

Reply:
Maybe "greater"?